arcgis-ios-sdk-samples/Maps/Display a map SwiftUI/DisplayMapSwiftUIView.swift

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import SwiftUI
+import ArcGIS
+
+struct DisplayMapSwiftUIView: View {
+    /// A boolean that indicates whether the action sheet is presented.
+    @State private var showingOptions = false
+    /// An object used for setting up a `SwiftUIMapView`.
+    @ObservedObject private var mapViewContext = MapViewContext()
```

Comment:
iOS 13 doesn't support [`StateObject`](https://developer.apple.com/documentation/swiftui/stateobject) so I have to use [`ObservedObject`](https://developer.apple.com/documentation/swiftui/observedobject) here instead. Also please see my comment about `EnvironmentObject` in `SwiftUIMapView`.